File: 2058-concatenation-of-array/concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] result = new int[2*n];
        for(int i=0; i<n;i++){
            result[i] = nums[i];
        }
        int count = 0;
        for(int i=n; i<2*n; i++){
            result[i] = nums[count++];
        }
        return result;
    }
}

File: 2058-concatenation-of-array/concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        n = len(nums)
        result = [0] * (2 * n)
        for i in range(n):
            result[i] = nums[i]

        for i in range(n):
            result[i + n] = nums[i]
        return result

        